### DISCUSSION TITLE
Image priority not preloading the image

### DISCUSSION THREAD

**Discussion by ahakem** [vercel/next.js#27955] [Help] [answered]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

14

### What browser are you using?

Chrome

### What operating system are you using?

macOS, Windows

### How are you deploying your application?

AWS with docker

### Describe the Bug

when I've a photo above the fold and using `priority` prop to `<Image />` I got the preload link in the head when I inspect but it's not the case when I use 
` CURL https://www.eagleshare.com --head `
which mean it's a fake preload, also lighthouse google page speed seeing that it's not preloaded

### Expected Behavior

to get it the preloaded links when you use the command
` CURL https://www.eagleshare.com --head`

### To Reproduce

CURL https://www.eagleshare.com --head
or check this domain on google page speed and check the inspecting source for the preload

**Comment by styfle** [comment, marked as answer]
> I got the preload link in the head 

This is the expected behavior, you should see the `<link>` in the `<head>` element like this: https://github.com/vercel/next.js/blob/eb871d30915d668dd9ba897d4d04ced207ce2e6d/packages/next/client/image.tsx#L633-L647

However, you won't see anything with `curl --head` because that only returns headers, not HTML `<head>`.

Try using `curl -i https://example.com` if you want to see both headers and the body of the response.
